Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```

Comment:
I'm not sure whether this change is inadvertent, or a workaround for something (perhaps the Rubygems SSL change?), or ???

But, I don't think it's appropriate to remove SSL protection, is it?